File: app/src/main/java/com/tutv/android/ui/profile/ProfileView.java
package com.tutv.android.ui.profile;

public interface ProfileView {
    void showToast(String text);

    void setProfileImage(String imageBase64);

    void setUsername(String text);

    void setMail(String text);

    void openProfileInWebApp(int userId);

    void setLoading(boolean loading);

    void setLayout(boolean isLoggedIn);
}

File: app/src/main/java/com/tutv/android/ui/profile/ProfileView.kt
package com.tutv.android.ui.profile

interface ProfileView {
    fun showToast(text: String?)
    fun setProfileImage(imageBase64: String?)
    fun setUsername(text: String?)
    fun setMail(text: String?)
    fun openProfileInWebApp(userId: Int?)
    fun setLoading(loading: Boolean)
    fun setLayout(isLoggedIn: Boolean)
}